src/sentry/rules/processing/delayed_processing.py

```diff
@@ -378,6 +378,20 @@ def get_group_to_groupevent(
     return group_to_groupevent
 
 
+def bucket_num_groups(num_groups: int) -> str:
+    if num_groups > 10000:
+        return ">10000"
+    elif num_groups > 1000:
+        return ">1000"
+    elif num_groups > 100:
+        return ">100"
+    elif num_groups > 10:
+        return ">10"
+    elif num_groups > 1:
+        return ">1"
+    return "1"
```

Comment:
can simplify the logic for 1 -> Infinity with: `10 ** int(math.log10(num_groups))` (ty dan) -- that will return the magnitude of the number.

to mimic what you have:
```suggestion
  if num_groups > 1:
    magnitude = 10 ** int(math.log10(num_groups))
    return f">{magnitude}"
  return "1"
```